crates/modelardb_server/src/storage/compressed_data_manager.rs

```diff
@@ -933,8 +933,8 @@ mod tests {
                 test::MODEL_TABLE_NAME.to_owned(),
                 query_schema,
                 vec![
-                    ErrorBound::try_new(0.0).unwrap(),
-                    ErrorBound::try_new(0.0).unwrap(),
+                    ErrorBound::try_new_relative(0.0).unwrap(),
```

Comment:
Fixed. However, even if the constants were public they could not be imported here, probably because they are defined in the module, so `ERROR_BOUND_ZERO` is also defined here.